### PR TITLE
Create API methods for manipulating with block elements

### PR DESCRIPTION
**Pull request by Darginec05**
This PR about:
- Creating `getElement, createElement, deleteElement`
- Simplifying API for manipulating with block elements
- Create `@yoopta/accordion` package
- Added new API's `Elements` and `Blocks`

### CODE REVIEW

File: packages/accordion/src/plugin/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  type: 'Accordion',
+  elements: {
+    'accordion-list': {
+      asRoot: true,
+      render: AccordionList,
+      children: ['accordion-list-item'],
+    },
+    'accordion-list-item': {
+      render: AccordionItem,
+      children: ['accordion-list-item-heading', 'accordion-list-item-content'],
```
Comment (maZahaca) [inline packages/accordion/src/plugin/index.tsx:18]: wondering an issue if you mistype the 'key', or if they will be the same within one plugin for multiple items

File: packages/accordion/src/renders/AccordionItem.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { PluginElementRenderProps } from '@yoopta/editor';
+
+export const AccordionItem = (props: PluginElementRenderProps) => {
+  const { element, attributes, children } = props;
+
+  return (
+    <li
+      {...attributes}
+      className="yoo-accordion-border yoo-accordion-border-[#525252] yoo-accordion-bg-[#303030] yoo-accordion-rounded-none yoo-accordion-relative yoo-accordion-w-full"
```
Comment (maZahaca) [inline packages/accordion/src/renders/AccordionItem.tsx:9]: wondering those two `[#525252]` is a standard coloring schema for yoopta, or/and might be re-used from somewhere or taken from Props with default values as defined in the code?

File: packages/accordion/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# `yoopta-accordion`
+
+> TODO: description
```
Comment (maZahaca) [inline packages/accordion/README.md:3]: Wondering if README is mandatory file for publishing the plugin to npm, it's a bit misleading.

File: packages/accordion/src/styles.css [hunk 4]
```diff
@@ -0,0 +1 @@
+@tailwind utilities;
```
Comment (maZahaca) [inline packages/accordion/src/styles.css:1]: \n

File: packages/accordion/src/types.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { SlateElement } from '@yoopta/editor';
+
+export type AccordionElementKeys =
```
Comment (maZahaca) [inline packages/accordion/src/types.ts:3]: What do you think about using enum here, those you can reuse and reference to it in other places like [here](https://github.com/Darginec05/Yoopta-Editor/pull/135/files#diff-37eadcf5c6abf12de5772c03e095efe7a5db9ea28f84160311cf9413fb372278R11-R16)

File: packages/core/src/editor/transforms/createBlock.ts [hunk 6]
```diff
@@ -27,6 +27,9 @@ export function createBlock(editor: YooEditor, type: string, options?: CreateBlo
   if (!rootBlockElement) return;
 
   const nodeProps = { nodeType: rootBlockElement.props?.nodeType || 'block', ...rootBlockElement.props };
+
+  console.log('nodeProps', nodeProps);
```
Comment (maZahaca) [inline packages/core/src/editor/transforms/createBlock.ts:31]: to be removed

File: packages/core/src/editor/types.ts [hunk 7]
```diff
@@ -59,12 +60,23 @@ export type YooptaBlock = {
...
-  delete: (options: DeleteBlockOptions) => void;
+  createElement: <TElementKeys extends string, TElementProps>(
+    blockId: string,
+    elementType: TElementKeys,
+    elementProps?: TElementProps,
+    options?: CreateBlockElementOptions,
+  ) => void;
+  getElement: <TElementKeys extends string>(
+    blockId: string,
+    elementType: TElementKeys,
```
Comment (maZahaca) [inline packages/core/src/editor/types.ts:77]: Do we essentially need to know elementType to get it by id?

File: packages/development/src/components/customPlugins/Accordion/Accordion.tsx [hunk 8]
```diff
@@ -0,0 +1,66 @@
+import { generateId, SlateElement, YooptaBlockData } from '@yoopta/editor';
```
Comment (maZahaca) [inline packages/development/src/components/customPlugins/Accordion/Accordion.tsx:1]: I have a very strong feeling that `example` should be stored within plugin and be generated to the website by Lerna, e.g.

in your plugin you have:
```
./README.md - docs
./test - tests folder
./example/
```

Also it gives a bit more understanding to the person who looks into the plugin repo of how to use it.

File: packages/core/src/editor/elements/getElement.ts [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+  const block = editor.children[blockId];
+
+  if (!block) {
+    throw new Error(`Block with id ${blockId} not found`);
+  }
+
+  const slate = findSlateBySelectionPath(editor, { at: [block.meta.order] });
+
+  if (!slate) {
+    console.warn('No slate found');
```
Comment (maZahaca) [inline packages/core/src/editor/elements/getElement.ts:19]: Is this really what we need do here (just warning), or trigger an Error?

File: packages/core/src/editor/elements/deleteElement.ts [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+  const block = editor.children[blockId];
+
+  if (!block) {
+    throw new Error(`Block with id ${blockId} not found`);
+  }
+
+  const slate = findSlateBySelectionPath(editor, { at: [block.meta.order] });
+
+  if (!slate) {
+    console.warn('No slate found');
```
Comment (maZahaca) [inline packages/core/src/editor/elements/deleteElement.ts:19]: maybe error?

**Comment by maZahaca** [comment]
In overall I think we can easily split this PR to smaller chunks (worth in future):
1. api changes
2. new plugin

File: packages/accordion/src/react-svg.d.ts [hunk 11]
```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
```
[the remark this answers is not in the document]
Comment (Darginec05) [reply packages/accordion/src/react-svg.d.ts]: to make svg icons working with typescript

File: packages/core/src/editor/types.ts (hunk 7 above)
Comment (Darginec05) [reply packages/core/src/editor/types.ts]: changed

**Pull request merged**